GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -1,14 +1,64 @@
 /**
- * This DebuggerClient connects to a websocket server and can dump
+ * An client side implementation of the Debugger
+ * @interface
+ */
+gdjs.DebuggerClient = function() {} // Necessary for Interface definition 
+
+/**
+ * Update a value, specified by a path starting from the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#set
+ * @param {Array<string>} path - The path of the property to modify, starting from the RuntimeGame.
+ * @param {any} newValue - The new value.
+ */
+
+/**
+ * Call a method, specified by a path starting from the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#call
+ * @param {Array<string>} path - The path to the method, starting from the RuntimeGame.
+ * @param {Array<any>} args - The arguments to pass the method.
+ */
+
+/**
+ * Dump all the relevant data from the {@link RuntimeGame} instance and send it to the server.
+ * @function
+ * @name gdjs.DebuggerClient#sendRuntimeGameDump
+ */
+
+/**
+ * Start profiling.
+ * @function
+ * @name gdjs.DebuggerClient#sendProfilerStarted
+ */
+
+/**
+ * Stop profiling.
+ * @function
+ * @name gdjs.DebuggerClient#sendProfilerStopped
+ */
+
+/**
+ * Send profiling results.
+ * @function
+ * @param {any} framesAverageMeasures The measures made for each frames.
+ * @param {any} stats Other measures done during the profiler run.
+ * @name gdjs.DebuggerClient#sendProfilerOutput
+ */
+
+
+/**
+ * This {@link DebuggerClient} connects to a websocket server, can dump
  * the data of the current game, and receive message to change a field or
- * call a function on an object of the specified runtimeGame.
+ * call a function, specified by a path from the {@link RuntimeGame}.
  *
  * @memberof gdjs
+ * @implements {gdjs.DebuggerClient}
  * @class WebsocketDebuggerClient
- * @param {gdjs.RuntimeGame} The `gdjs.RuntimeGame` to be debug
+ * @param {gdjs.RuntimeGame} runtimeGame -  The `gdjs.RuntimeGame` to be debugged
  */
-gdjs.WebsocketDebuggerClient = function(runtimegame) {
-  this._runtimegame = runtimegame;
+gdjs.WebsocketDebuggerClient = function(runtimeGame) {
+  this._runtimegame = runtimeGame;
```

Comment:
The rest of the function was using `runtimegame` variable
Always be sure to check the working of what you change :) 
Tip: if this case you can use a multi cursor (Ctrl/Cmd+D on Sublime/Atom/VSCode) to select all the occurence of runtimegame, so that you're sure that you don't forget anything.

Reply:
This is also the reason why I was pushing for `// @ts-check`. 
This is not your fault (I also missed this while reading the code ;)), this should have been caught by an automated tool. Life is too short to lose time with this silly stuff. Having more files with `// @ts-check` will help us make sure the code has no obvious bugs like this.